apps/ui/lib/models/phone/Phone.mjs

```diff
@@ -0,0 +1,16 @@
+import { phone } from 'phone';
+
+export class Phone {
+    constructor(phoneNumber, countries) {
+        this.phoneNumber = phoneNumber;
+        this.validated = Phone.isValid(this.phoneNumber, countries);
+    }
+
+    static isValid(phoneNumber, countries) {
```

Comment:
you dont need to pass countries here as an argument. as i understand the list is hardcoded. it could be a part of Phone class